collect-cards-with-individual-elements-react/src/CheckoutForm.tsx

```diff
@@ -58,12 +58,12 @@ export const CheckoutForm = (): JSX.Element => {
 
   const style: ElementStyle = {
     fonts: [
-      'https://fonts.googleapis.com/css2?family=Source+Sans+Pro:ital,wght@0,200;0,300;0,400;0,600;0,700;0,900;1,200;1,300;1,400;1,600;1,700;1,900&display=swap',
+      'https://fonts.googleapis.com/css2?family=Inter:ital,wght@0,200;0,300;0,400;0,600;0,700;0,900;1,200;1,300;1,400;1,600;1,700;1,900&display=swap',
     ],
     base: {
       color: '#fff',
       fontWeight: 500,
-      fontFamily: "'Source Sans Pro'",
+      fontFamily: "'Inter'",
```

Comment:
```suggestion
      fontFamily: '"Inter"',
```

Just for consistency.

Reply:
YES!